src/math/CompoundMath.sol

```diff
@@ -8,7 +8,7 @@ pragma solidity ^0.8.0;
 library CompoundMath {
     /// CONSTANTS ///
 
-    uint256 internal constant MAX_UINT256 = 2**256 - 1;
+    uint256 internal constant MAX = 2**256 - 1;
     uint256 public constant SCALE = 1e36;
```

Comment:
Unrelated, but maybe we should be declaring these as internal too. Declaring a library variable as public would technically make this an external library.